build-system/babel-plugins/babel-plugin-transform-rename-privates/index.js

```diff
@@ -9,6 +9,14 @@
 module.exports = function (babel) {
   const {types: t} = babel;
 
+  /**
+   * @param {string} name
+   * @return {boolean}
+   */
+  function isRenamed(name) {
+    return name.endsWith('_AMP_PRIVATE_') || name.endsWith('_AMP_ENUM_KEY_');
```

Comment:
were some variables being renamed twice? how?

Reply:
Because `ENUM_KEY_` ends in a `_`, which means it could have the private renamer applied.